engine/apps/schedules/quality_score.py

```diff
@@ -22,17 +22,27 @@ def get_schedule_quality_score(events: list[dict], days: int) -> dict:
     else:
         total_score = 0
 
-    gaps_comment = "Schedule has gaps" if good_event_score < 1 else "Schedule has no gaps"
+    info_comments = []
+    warning_comments = []
+
+    if good_event_score < 1:
+        warning_comments.append("Schedule has gaps")
+    else:
+        info_comments.append("Schedule has no gaps")
+
     if balance_score < 0.8:
-        balance_comment = "Schedule has balance issues"
+        warning_comments.append("Schedule has balance issues")
     elif 0.8 <= balance_score < 1:
-        balance_comment = "Schedule is well-balanced, but still can be improved"
+        info_comments.append("Schedule is well-balanced, but still can be improved")
     else:
-        balance_comment = "Schedule is perfectly balanced"
+        info_comments.append("Schedule is perfectly balanced")
+
+    comments = [{"type": "warning", "text": comment} for comment in warning_comments]
```

Comment:
maybe define some constant values for "warning" and "info" strings? also wondering if directly building the `comments` value would look too complex? (since it seems we don't need to have the intermediate lists?)

Reply:
Good call! Updated in https://github.com/grafana/oncall/pull/1461/commits/1e2faef4fba853fd05a01a77740261cc0297080f